README.md

```diff
@@ -120,15 +162,21 @@ And we wish to rollback production to `1.0.4` then:
 npm dist-tag add my-module@1.0.4 prod
 ```
 
-This will trigger a build of `my-module@1.0.4` (since it's dependencies may have changed since 1.0.4 was latest in production) and a build of all modules that depend on `my-module`.
+This will trigger a build of `my-module@1.0.4` (since it's dependencies may
+have changed since 1.0.4 was latest in production) and a build of all modules
+that depend on `my-module`.
```

Comment:
This paragraph isn't really accurate (can be updated later since you're just touching formatting). but if 1.0.4 was in prod before, it'll just reuse the same build as last time.